packages/editor/src/components/post-locked-modal/index.js

```diff
@@ -30,17 +30,30 @@ class PostLockedModal extends Component {
 	}
 
 	componentDidMount() {
+		const hookNamePrefix = this.getHookNamePrefix();
+
 		// Details on these events on the Heartbeat API docs
 		// https://developer.wordpress.org/plugins/javascript/heartbeat-api/
-		jQuery( document )
-			.on( 'heartbeat-send.refresh-lock', this.sendPostLock )
-			.on( 'heartbeat-tick.refresh-lock', this.receivePostLock );
+		addAction( 'heartbeat.send', hookNamePrefix + '-send', this.sendPostLock );
```

Comment:
As I mentioned [here](https://github.com/WordPress/gutenberg/pull/11781/files#r237470204), how about:

```suggestion
		addAction( 'heartbeat.send', hookNamePrefix, this.sendPostLock );
		addAction( 'heartbeat.tick', hookNamePrefix, this.sendPostLock );
```

Reply:
I guess, the Suggested change beta feature doesn't like multicomment lines :/